### PR TITLE
RavenDB-21833 fix CheckServerVersion

### PR DESCRIPTION
**Pull request by efratshenhar**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-21833

### Additional description

fix CheckServerVersion to support v6.1

### Type of change

- [x] Bug fix
- [ ] Regression bug fix
- [ ] Optimization
- [ ] New feature

### How risky is the change?

- [x] Low 
- [ ] Moderate 
- [ ] High
- [ ] Not relevant

### Backward compatibility

- [x] Non breaking change
- [ ] Ensured. Please explain how has it been implemented?
- [ ] Breaking change
- [ ] Not relevant

### Is it platform specific issue?

- [ ] Yes. Please list the affected platforms.
- [x] No

### Documentation update

- [ ] This change requires a documentation update. Please mark the issue on YouTrack using `Documentation Required` tag.
- [x] No documentation update is needed 

### Testing by Contributor

- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] Internal classes added to the test class (e.g. entity or index definition classes) have the lowest possible access modifier (preferable `private`) 
- [x] It has been verified by manual testing

### Testing by RavenDB QA team

- [ ] This change requires a special QA testing due to possible performance or resources usage implications (CPU, memory, IO). Please mark the issue on YouTrack using `QA Required` tag.
- [x] No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- [ ] Yes. Please list the affected features/subsystems and provide appropriate explanation
- [x] No

### UI work

- [ ] It requires further work in the Studio. Please mark the issue on YouTrack using `Studio Required` tag.
- [ ] No UI work is needed

### CODE REVIEW

File: src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs [hunk 1]
```diff
@@ -281,8 +281,11 @@ private static bool CheckServerVersion(string serverVersion)
             if (serverVersion != null && Version.TryParse(serverVersion, out var ver))
             {
                 // Version 6 only from 6.0.2
+                if (ver.Major == 6 && ver.Minor > 0)
```
Comment (ppekrol) [inline src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs:284]: this need to go to v6.0 branch

File: src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs [hunk 2]
```diff
@@ -281,8 +281,11 @@ private static bool CheckServerVersion(string serverVersion)
...
+                    return true;
                 if (ver.Major == 6 && ver.Build < 2)
```
Comment (ppekrol) [inline src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs]: be more explicit here with the version, check also that Minor == 0

File: src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs (hunk 1 above)
Comment (efratshenhar) [reply src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs:284]: Done

File: src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs (hunk 2 above)
Comment (efratshenhar) [reply src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs]: Done

**Review by arekpalinski — APPROVED** [review]

**Pull request merged**